graylog2-web-interface/src/components/indices/IndicesPageNavigation.tsx

```diff
@@ -22,7 +22,12 @@ import Routes from 'routing/Routes';
 import { Row } from 'components/bootstrap';
 
 const PREM_ONLY_NAV_ITEMS = [
-  { description: 'Index Set Templates', path: Routes.SYSTEM.INDICES.TEMPLATES.OVERVIEW, exactPathMatch: false },
+  {
+    description: 'Index Set Templates',
+    path: Routes.SYSTEM.INDICES.TEMPLATES.OVERVIEW,
+    exactPathMatch: false,
+    permissions: 'indexset_templates:read',
```

Comment:
@laura-b-g I'm not sure if this is what you had in mind for this but it seems like this was the existing built in method to add permissions for nav items so I went for it.